results.tex

```diff
@@ -0,0 +1,143 @@
+\section{Results}
+
+\subsection{Past-Predictable Future Scenario}
+
+\ref{tab:sim_result} lists the important metrics
+obtained from the first simulation. The following
+values are the EU inventory and history at year 2050.
+
+\begin{table}[h]
+	\centering
+	\label{tab:sim_result}
+	\caption {Simulation Results}
+	\scalebox{0.86}{
+		\begin{tabular}{|c|c|c|}
+			\hline
+			Category [unit] & Value & Specifics \\ \hline
+			Total UOX Usage [t] & 188,196  &  \\ \hline
+			Total MOX Usage [t] & 118 & \\ \hline
+			Total Spent UOX [t] & 183,807 & \\ \hline
+			Total Spent MOX [t] & 0 & All is Reprocessed. \\ \hline
+			Total Tailings [t] & 1,125,798 & \\ \hline
+		\end{tabular}}
+\end {table}
+
+\begin{table}[h]
+	\centering
+	\label{tab:pu}
+	\caption{Plutonium From Spent Fuel}
+	\begin{tabular}{|c|c|c|}
+		\hline
+		Isotope & Mass Fraction in Spent Fuel [\%] & Quantity [t] \\ \hline
+		Total & .9358 & 1,761.13 \\ \hline
+		Pu238 & .0111 & 20.88 \\ \
+		Pu239 & .518 & 974.85 \\ \hline
+		Pu240 & .232 & 436.61 \\ \hline
+		Pu241 & .126 & 237.12 \\ \hline
+		Pu242 & .0487 & 91.65 \\ \hline
+	\end{tabular}
+\end{table}
+
+
+To create MOX, 10\% Pu and 90\% depleted uranium is used.
+Thus $1,761.13$ tons of plutonium yields $17,611.3$ tons of
```

Comment:
That's a few more significant figures than I think is probably appropriate here.